dll/ntdll/CMakeLists.txt

```diff
@@ -60,10 +60,6 @@ set_module_type(ntdll win32dll ENTRYPOINT 0)
 set_subsystem(ntdll console)
 ################# END  HACK #################
 
-if(MSVC)
-    add_target_link_flags(ntdll "/RELEASE")
-endif()
```

Comment:
This is non-obvious in a commit that just claims to replace the command used to add linker flags.
Is this redundant because we add it by default, or... ?

Reply:
The commit that added this extra `/RELEASE` flag: https://github.com/reactos/reactos/commit/005a7acbfcf8dc2fb74944dc39882084a95740c7 , claimed it's for getting a PE checksum.
However in the meantime, for MSVC builds, this flag is systematically added, see https://github.com/reactos/reactos/blob/master/sdk/cmake/msvc.cmake#L177 , so it may be fine to remove it there, indeed.

Reply:
Yeah, `/RELEASE` is added globally, so I decided to remove the flag here